print-domain.c

```diff
@@ -596,6 +597,15 @@ ns_rprint(netdissect_options *ndo,
 		ND_PRINT(" otherlen=%u", GET_BE_U_2(cp));
 		cp += 2;
 	    }
+
+	case T_URI:
+		if (!ND_TTEST_LEN(cp, len))
+			return(NULL);
+		ND_PRINT(" %u %u ", EXTRACT_BE_U_2(cp), EXTRACT_BE_U_2(cp + 2));
```

Comment:
Following a recent change (176e182416d822df0f9d4695410479e8b17a07b3 and ee68aa36460d7efeca48747f33b7f2adc0900bfb), you need to use GET_ macros instead of EXTRACT_ ones.

Reply:
Thanks for the review. I have force-pushed code with the needed change and fix of a careless switch case fallthrough. 

But the CI insists that conflict exists, and I don't have clue how to deal with it now. Can you take a look at it?